tests/run-build.sh

```diff
@@ -10,52 +10,65 @@ if test -e ci-skip ; then
     exit 0
 fi
 
-# Setup go environment on semaphore
-if [ -f /opt/change-go-version.sh ]; then
-    . /opt/change-go-version.sh
-    change-go-version 1.4
-fi
-
+SRC_CHANGES=1 # run functional tests by default
 RKT_STAGE1_USR_FROM="${1}"
 RKT_STAGE1_SYSTEMD_VER="${2}"
-
 BUILD_DIR="build-rkt-${RKT_STAGE1_USR_FROM}-${RKT_STAGE1_SYSTEMD_VER}"
 
+if [ "${CI-}" == true ] ; then
+	# https://semaphoreci.com/docs/available-environment-variables.html
+	if [ "${SEMAPHORE-}" == true ] ; then
+        # We might not need to run functional tests
+        SRC_CHANGES=$(git diff-tree --no-commit-id --name-only -r HEAD..origin/master | \
+            grep -vE "^Documentation/" | wc -l)
+
+        # Setup go environment on semaphore
+        if [ -f /opt/change-go-version.sh ]; then
+            . /opt/change-go-version.sh
+            change-go-version 1.4
+        fi
+
+        # Semaphore does not clean git subtrees between each build.
+        sudo rm -rf "${BUILD_DIR}"
```

Comment:
We clone to ${BUILD_DIR} regardless, so we have to remove it regardless. The `rm` from L32 is because we assume that semaphore didn't cleanup after itself. Basically, we should just check if it's there at all and remove it before cloning it.